etc/webapi.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<routes xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../../../app/code/Magento/Webapi/etc/webapi.xsd">
```

Comment:
@NandhiniKarthi Update the urn similar to other xmls. this sometimes picksup wrongly due to relative referencing.